draft-ietf-dprive-dnsoquic.md

```diff
@@ -54,7 +54,7 @@ informative:
 
 --- abstract
 
-This document describes the use of QUIC to provide transport privacy for DNS.
```

Comment:
"privacy", like "security" is a complex and multifaceted topic, and trying to distill any discussion down to just the single word renders it largely meaningless.  I propose to use a more precise word here in the abstract where we care the most about concision; in the body we mostly discuss "privacy protection" and in analogy with DoT, which doesn't seem as problematic.

Reply:
Yes, confidentiality is a better word there.